syftlib/src/main/java/org/openmined/syft/networking/clients/SyftWebSocket.kt

```diff
@@ -11,30 +11,58 @@ import okhttp3.WebSocketListener
 import org.openmined.syft.networking.requests.NetworkingProtocol
 import java.util.concurrent.TimeUnit
 
-
+// Used in serializing data to be passed over the network
 const val TYPE = "type"
 const val DATA = "data"
+// Code used to close web socket connection
 private const val SOCKET_CLOSE_CLIENT = 1000
 
+/**
+ * SyftWebSocket initialize and configure Web Socket connection
+ * @property protocol {@link org.openmined.syft.networking.requests.NetworkingProtocol NetworkingProtocol} to be used
```

Comment:
You are right here. SyftWebSocket doesn't have these values as properties and it makes sense to mark them as params here